README.md

```diff
@@ -1,57 +1,22 @@
-# Node.js Internationalization
+# Node.js i18n
 
 [![Crowdin](https://badges.crowdin.net/nodejs/localized.svg)](https://crowdin.com/project/nodejs)
 
+This repo is a basement of Node.js projects internationalization initiative. Here you can find contribution guides, discuss ideas or join meetings. Here also contains original and translated Node.js API documentation.
```

Comment:
'basement' is not idiomatic here (though I understand your meaning), I think you can just delete this sentence because it's in the opening line:

~This repo is a basement of Node.js projects internationalization initiative.~